projects/netgrif-components/src/lib/navigation/quick-panel/components/impersonate-quick-panel/impersonate-quick-panel.component.spec.ts

```diff
@@ -0,0 +1,53 @@
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+
+import {ImpersonateQuickPanelComponent} from './impersonate-quick-panel.component';
+import {
+    AuthenticationMethodService,
+    ConfigurationService,
+    MaterialModule,
+    MockAuthenticationMethodService,
+    SnackBarService,
+    TranslateLibModule
+} from 'netgrif-components-core';
+import {
+    NaeExampleAppConfigurationService
+} from '../../../../../../../nae-example-app/src/app/nae-example-app-configuration.service';
```

Comment:
there shouldnt be an import from example app , for test use test configuration service from NCC

Reply:
done